### PR TITLE
Fix to fail grpc connection when configuration fails #1526

### PR DESCRIPTION
**Pull request by tommy5oo**

### CODE REVIEW

File: app/src/main/java/com/skt/nugu/sampleapp/activity/MainActivity.kt [hunk 1]
```diff
@@ -409,42 +409,39 @@ class MainActivity : AppCompatActivity(), SpeechRecognizerAggregatorInterface.On
                 ConnectionStatusListener.ChangedReason.SERVER_SIDE_DISCONNECT
                 -> {
                     /**
-                     * only server-initiative-directive
                      * If you want to reconnect to the server, run the code below.
                      * But it can be recursive, so you need to manage the count of attempts.
                      **/
-                    if (NuguOAuth.getClient().isSidSupported()) {
```
Comment (shshshl09) [inline app/src/main/java/com/skt/nugu/sampleapp/activity/MainActivity.kt:416]: why this condition removed? no need anymore?

Comment (tommy5oo) [reply app/src/main/java/com/skt/nugu/sampleapp/activity/MainActivity.kt:416]: Previously, the connection did not fail by applying the default address, but applying the configurationMetadata(nullable) may cause the connection to fail.

Comment (tommy5oo) [reply app/src/main/java/com/skt/nugu/sampleapp/activity/MainActivity.kt:416]: See code below, metadata can be null. 
```
.transportFactory(
                GrpcTransportFactory(NuguServerInfo(object : NuguServerInfo.Delegate {
                    override fun getNuguServerInfo() : NuguServerInfo? {
                        val metadata = ConfigurationStore.configurationMetadataSync()
                        return NuguServerInfo.Builder().deviceGW(metadata?.deviceGatewayServerGrpcUri)
                            .registry(metadata?.deviceGatewayRegistryUri)
                            .keepConnection(NuguOAuth.getClient().isSidSupported())
                            .build()
                    }
                }))
            )
```

**Review by shshshl09 — APPROVED** [review]

File: nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt [hunk 2]
```diff
@@ -111,22 +115,41 @@ class ChromeWindowContentLayout @JvmOverloads constructor(
...
+            isPendingDismiss.set(true)
+        }
     }
 
     fun expand() {
-        bottomSheetBehavior.state = BottomSheetBehavior.STATE_EXPANDED
+        if(!isPendingDismiss.get()) {
+            bottomSheetBehavior.state = BottomSheetBehavior.STATE_EXPANDED
+        }
+        isPendingExpand.set(true)
```
Comment (Yuseon-Han) [inline nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt:152]: why isPendingExpand always become true?

**Review by Yuseon-Han — APPROVED** [review]

File: nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt (hunk 2 above)
Comment (tommy5oo) [reply nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt:152]: Right. I'll fix it.

Comment (tommy5oo) [reply nugu-ux-kit/src/main/java/com/skt/nugu/sdk/platform/android/ux/widget/ChromeWindowContentLayout.kt]: fixed it

File: app/src/main/java/com/skt/nugu/sampleapp/activity/MainActivity.kt (hunk 1 above)
Comment (tommy5oo) [reply app/src/main/java/com/skt/nugu/sampleapp/activity/MainActivity.kt]: Improved this issue!

Comment (tommy5oo) [reply app/src/main/java/com/skt/nugu/sampleapp/activity/MainActivity.kt]: Fix to adjust the order of connect in non-SID

**Pull request merged**